manifests/x/xiles/NexusFont/2.7.1/xiles.NexusFont.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: xiles.NexusFont
+PackageVersion: 2.7.1
+MinimumOSVersion: 10.0.0.0
+Installers:
+- Architecture: x86

```

Comment:
```suggestion
- Architecture: x64
```

Reply:
Hi @14AUDDIN,

`x86` packages are validated on a 32-bit OS, so I'm guessing this is a 64-bit application.